### ISSUE TITLE
ArgumentError using extract_all_marc_values

### ISSUE THREAD

**Issue by jcoyne** [traject/traject#105]
My config looks like this:
```
to_field "text", extract_all_marc_values(:from => 100, :to => 899)
```

```
2015-06-11T19:02:19-05:00  INFO traject (2.0.2) executing with: `-c traject_config.rb spec/fixtures/marc/adrl_etd_sample.mrc`
2015-06-11T19:02:19-05:00  INFO Reading from spec/fixtures/marc/adrl_etd_sample.mrc
2015-06-11T19:02:19-05:00  INFO    Indexer with 1 processing threads, reader: Traject::MarcReader and writer: Traject::JsonWriter
2015-06-11T19:02:19-05:00 ERROR Unexpected error on record id `` at file position 1
    while executing (to_field text at traject_config.rb:11)
    Exception: ArgumentError: comparison of String with 100 failed
    /Users/justin/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/traject-2.0.2/lib/traject/macros/marc21.rb:173:in `>='
```

**Comment by billdueber** [comment]
It's expecting strings (e.g., "100" and "899"). I'll add some code at https://github.com/traject/traject/blob/be16556c752f6d97d21cc200f2f3fa11280864a4/lib/traject/macros/marc21.rb#L169 to check and see if from/to are fixnums and cast them to strings, and also update the README.

**Comment by jcoyne** [comment]
Yes, I was following directions from the readme

**Comment by billdueber** [comment]
Fixed by #125
